Release/include/cpprest/ws_msg.h

```diff
@@ -73,6 +73,15 @@ class websocket_outgoing_message
 {
 public:
 
+	/// <summary>
+	/// Sets a the outgoing message to be an unsolicited pong message.
+	/// This is useful when the client side wants to check whether the server is alive.
+	/// </summary>
+	void set_pong_message()
```

Comment:
This will fail on WinRT with an exception "Message Type not supported.".
Another option is to not expose the API on WinRT.